scanner/src/main/kotlin/Scanner.kt

```diff
@@ -177,8 +177,7 @@ class Scanner(
     }
 
     suspend fun scan(packages: Set<Package>, context: ScanContext): ScannerRun {
-        val scanners = scannerWrappers[context.packageType].orEmpty()
-        if (scanners.isEmpty()) return ScannerRun.EMPTY
+        val scanners = scannerWrappers[context.packageType]?.takeIf { it.isNotEmpty() } ?: return ScannerRun.EMPTY
```

Comment:
Personally, I actually find the new code harder to read. However, how about

        val scanners = scannerWrappers[context.packageType]
        if (scanners.isNullOrEmpty()) return ScannerRun.EMPTY